add-zeroes-with-variable

```diff
@@ -0,0 +1,13 @@
+print("Sudoku Generator Starting Up!")
+
+# one for each character position, space margins around them,
+# and border spots for pipes.
+horizontal_cell_count = 9
+text_cells_in_ascii_grid = (4 * horizontal_cell_count + 1)
+number_in_cell = 0
```

Comment:
This one is even better! Because it defines the "data" in one place, with this variable, and produces the presentation in a different place , the loop below.  Notice that because this file doesn't have ".py" at the end, github treats is as just text, whereas the other file it shows the changes to and gives some nice syntax highlighting for python code.

I would go ahead and replace the code in "sudoku_generator.py" with what you've got in this "add-zeroes-with-variable" file, and delete this file afterwards.  The history of every change we ever made will still be in the repository if we want to go back and look at it later.  Nice work!